hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecord.java

```diff
@@ -133,6 +133,11 @@ public String getFieldName() {
    */
   protected HoodieRecordLocation newLocation;
 
+  /**
+   * If set, not update index after written.
+   */
+  protected boolean ignored;
```

Comment:
may be we can call it as `ignoreIndexUpdate` 
and method can be named as "canIgnoreIndexUpdate"

Reply:
@bvaradar : is this addressed ?